write-fonts/src/tables/gvar.rs

```diff
@@ -325,22 +325,57 @@ impl GlyphDelta {
     }
 }
 
+/// The influence of a single axis on a variation region.
+///
+/// The values here end up serialized in the peak/start/end tuples in the
+/// [`TupleVariationHeader`].
+///
+/// The name 'Tent' is taken from HarfBuzz.
+#[derive(Clone, Debug, PartialEq, Eq, PartialOrd, Ord, Hash)]
+#[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
+pub struct Tent {
+    peak: F2Dot14,
+    min: F2Dot14,
```

Comment:
this is probably just cosmetic, as this isn't really a tuple but a struct with named fields, but I'm used to seeing a "tent" as a { min, peak, max } in this particular order, where peak is between min and max, by definition. I wonder if we should reoder.
Also about min/max.. I believe the spec prefers to call them start and end respectively, though the meaning is clear anyway.

Reply:
we're using min/max elsewhere, so I'm happy to just keep it consistent. ordering is an internal detail so we can change that later, I think it makes sense to get this merged. :)